### PR TITLE
Contact Form Project

### PR DESCRIPTION
**Pull request by Josphat205**
These are the changes that I made in the project:

- Added I form inputs of type text, email, and text-area
- Added method to form 
- Added action and copied the link from form-spree after creating an account
- Added a responsive button in my form
- Added placeholders in my input to direct my users

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -196,29 +196,35 @@ <h2>Multi-Post Stories</h2>
...
-             <li class="links"><a href="#"><ion-icon name="logo-twitter"></ion-icon></a></li>
-             <li class="links"><a href="#"><ion-icon name="logo-pinterest"></ion-icon></a></li>
-         </ul>
-         <p class="mail">Jloman200@gmail.com</p>
-         <img src="img/Shape.png" alt="" class="end">
-        </div>
+        <footer>
+            <section class="contact-section">
+            <div class="contact-title">
+                <h2>CONTACT US</h2>
```
Comment (thecodechaser) [inline index.html:202]: - [ ] Please use correct color for this heading to match with the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A2346)template `#9390ff`.

File: index.html [hunk 2]
```diff
@@ -196,29 +196,35 @@ <h2>Multi-Post Stories</h2>
...
+                <h2>CONTACT US</h2> 
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+            </div>
+            <form class="form-area" id="form_id" action="https://formspree.io/f/xrgjoprq" method="POST">
+                <!-- <label for="name">Name</label> -->
+                <input type="text" name="name" placeholder="Enter your name here" required maxlength="30">
+                <!-- <label for="email">Email</label> -->
+                <input type="email" name="email" placeholder="youremail@gmail.com" required>
+                <!-- <label for="message">Message</label> -->
+                <textarea name="message" placeholder="Enter the message here..." required maxlength="500"></textarea>
```
Comment (thecodechaser) [inline index.html:211]: - [ ] Please place some padding to form fields and use normal font-weight for the placeholders and make sure it matches with the Figma design.

### Your design

image

### Figma design

image

File: index.html [hunk 3]
```diff
@@ -196,29 +196,35 @@ <h2>Multi-Post Stories</h2>
...
+                <button type="submit"  class="sbtn">Send Message</button>
```
Comment (thecodechaser) [inline index.html:212]: - [ ] Please increase the top-bottom padding for this button, use normal font-weight for the text and remove the border radius and make sure it matches with the Figma design

### Your design

image

### Figma design

image

File: index.html [hunk 4]
```diff
@@ -196,29 +196,35 @@ <h2>Multi-Post Stories</h2>
...
+            </form>
+            </section>
+            <div class="nav-button">
+            <img src="img/v.png" alt="" class="line">
+            <ul class="media-links">
+                <li class="links"><a href="#"><ion-icon name="logo-github"></ion-icon></a></li>
+                <li class="links"><a href="#"><ion-icon name="logo-linkedin"></ion-icon></a></li>
+                <li class="links"><a href="#"><ion-icon name="logo-twitter"></ion-icon></a></li>
+                <li class="links"><a href="#"><ion-icon name="logo-whatsapp"></ion-icon></a></li>
+                <li class="links"><a href="#"><ion-icon name="logo-pinterest"></ion-icon></a></li>
```
Comment (thecodechaser) [inline index.html:222]: - [ ] Please increase the gap between these social media icons and increase icon size and make sure it matches with the Figma design.

### Your design

image

### Figma design
image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Josphat205, and @sameh080081👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Josphat205** [comment]
changes made:

- Added padding to form input,
- Increased font size to social links,
- Increased gap between social links,
- Increased padding to submit button and changed form weight,

**Review by AnselemOdims — APPROVED** [review]
### Hi  @Josphat205 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**